### PR TITLE
Use notBlank() for validation.

### PR DESCRIPTION
**Pull request by bcrowe**
Follows https://github.com/cakephp/cakephp/pull/6285

### CODE REVIEW

File: src/Template/Bake/Model/table.ctp [hunk 1]
```diff
@@ -119,7 +119,7 @@ foreach ($validation as $field => $rules):
                     $field
                 );
                 $validationMethods[] = sprintf(
-                    "->notEmpty('%s')",
+                    "->notBlank('%s')",
```
Comment (markstory) [inline src/Template/Bake/Model/table.ctp]: The validator still have `notEmpty()` though and no `notBlank()` method.

Comment (ADmad) [inline src/Template/Bake/Model/table.ctp]: `Validation::notBlank()` has been added to core's `master` branch.

Comment (ADmad) [inline src/Template/Bake/Model/table.ctp]: Err disregard my above comment. This `notEmpty()` is `Validator::notEmpty()` not `Validation::notEmpty()` so no change needs to be done.

Comment (bcrowe) [inline src/Template/Bake/Model/table.ctp]: Yeah, I totally misread the file name your PR. Eeeep. Sorry!

**Pull request closed without merging**